browser/main/modals/PreferencesModal/UiTab.js

```diff
@@ -293,7 +294,16 @@ class UiTab extends React.Component {
             </div>
             : null
           }
-
+          <div styleName='group-checkBoxSection'>
```

Comment:
I think you should move this config down to editor setting

Reply:
as you asked me I put the config in the editor setting